<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>DEfault</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_default"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_test"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="cnbEcho"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkReadOnly">
        <property name="text">
         <string>ReadOnly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkClearButton">
        <property name="text">
         <string>ClearButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
